app/services/metrics/send_metrics.rb

```diff
@@ -0,0 +1,26 @@
+module Metrics
+  class SendMetrics
+    PROMETHEUS_THREAD_SLEEP = 0.1
+
+    def self.call
+      new.call
+    end
+
+    def call
+      Metrics::SidekiqQueueSizes.call(prometheus_client)
+
+      # prometheus_exporter runs in a loop and we need to give it some time to send the metrics
+      # https://github.com/discourse/prometheus_exporter/blob/master/lib/prometheus_exporter/client.rb#L54
+      sleep PROMETHEUS_THREAD_SLEEP * 2
+    end
+
+    private
+
+    def prometheus_client
+      @prometheus_client ||= PrometheusExporter::Client.new(
+        host: Rails.configuration.x.service_host,
+        thread_sleep: PROMETHEUS_THREAD_SLEEP
+      )
+    end
```

Comment:
Right now, this class `SendMetrics` only calls `SidekiqQueueSizes` and I anticipate that it will send other metrics in the future with other service classes similar to `SidekiqQueueSizes` but using the same prometheus client.
But you're right, at the moment, this cache is useless so I'll remove it